public/index.html

```diff
@@ -40,7 +40,7 @@
 
     <script async defer data-domain="beta.yearn.finance" src="https://analytics.yearn.finance/js/plausible.js"></script>
   </head>
-  <body>
+  <body class='transition-colors duration-150 bg-background' data-theme='system-prefs'>
```

Comment:
The class could be removed, but: 
- `transition-colors duration-150` makes the transition between theme a bit smoother
-  `bg-background` overwrite the default background color (even on overscroll) by the one currently selected via the theming system

The `data-theme` argument is used to indicate the current theme and be able to play with it in the css/js. Currently, 3 themes are supported:
- `system-prefs` -> `dark` or `light` depending on the user's preference
- `dark` -> well, dark theme
- `light` -> yes, light theme

The custom themes are **NOT** supported by the lib yet, we should add them

Reply:
Can we try to add these styles directly in the global style [here](https://github.com/yearn/yearn-finance-v3/blob/develop/src/client/index.tsx#L14) ?

a reference from styled components docs [here](https://styled-components.com/docs/api#createglobalstyle)

Just to try to keep globally applied css in one place.

Reply:
Fixed with [5b1ee4e](https://github.com/yearn/yearn-finance-v3/pull/622/commits/5b1ee4e84b83871a45aff82c84a66a41a98b5aa1) ser!